pkg/ring/replication_strategy.go

```diff
@@ -49,6 +39,14 @@ func (s *defaultReplicationStrategy) Filter(instances []InstanceDesc, op Operati
 		}
 	}
 
+	// We need a response from a quorum of instances, which is n/2 + 1.  In the
+	// case of a node joining/leaving, the actual replica set might be bigger
```

Comment:
Can we say that this is specifically when extend-writes is on?  "might" is unnerving.